src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -465,12 +425,15 @@ def cli_cosmosdb_gremlin_graph_update(client,
                                           None):
         logger.debug('replacing Gremlin graph')
 
+    gremlin_graph_create_update_resource = GremlinGraphCreateUpdateParameters(
+        resource=gremlin_graph_resource,
+        options={})
```

Comment:
Why do you want to pass options with {} here?

Reply:
This is the same reasoning as with MongoDBCollectionCrateUpdateParameters. It is a require parameter in the SDK, but we do not allow option updates via this command.